content/explicit_rma_intro.tex

```diff
@@ -0,0 +1,12 @@
+The explicit nonblocking RMA routines provide interfaces for copying
+data from a contiguous local data object to a data object on a specified
+PE. The implicit and explicit nonblocking operations are similar in
+their semantics on return from the call .i.e., on return, they are not
+guaranteed to be completed either locally or remotely. However, they
+differ in their interfaces and how the operations are completed. The
```

Comment:
"However, they differ in their interfaces and how the operations are completed."
This sentence seems obvious. Please elaborate it or remove it.